frontend/app/templates/components/person-edit.hbs

```diff
@@ -151,13 +146,11 @@
           </div>
         </div>
         <br>
-        <button {{action "submit" personChangeset}} role="button" id="submit-button" class="btn btn-primary">
-          Speichern
-        </button>
+        {{personChangeset.submit label="Speichern" id="submit-button"}}
```

Comment:
Could you also add the bootstrap btn classes back to this? :)

Reply:
It should've the bootstrap btn classes since I'm using the bootstrap theme for the validation addon.

Reply:
I must've not executed yarn install or something. You're obviously right, the styling's completely fine.